### PR TITLE
Add doc on logit

### PR DESCRIPTION
**Pull request by surminus**
I included this in a previous PR, but I'd like to merge it early as I intend to sneakily redirect people to this document when they try to browse the old Kibana URL while testing.

https://trello.com/c/YrTOpoq7/746-logit-migration

I have also included some queries as I have given up trying to pass in a direct link to Icinga for Kibana 5/Logit.

### CODE REVIEW

File: source/manual/logit.html.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+right of the Chrome browser.
+
+Go to the bottom of the selection of apps and you should find one for "Logit".
+
+Clicking on this should log you in.
+
+### Adding your user to the right team
+
+After you have logged in, if you are unable to see any "stacks", please speak to
+a GOV.UK Logit administrator.
```
Comment (afda16) [inline source/manual/logit.html.md]: How can people find a GOV.UK Logit Administrator?

Comment (surminus) [reply source/manual/logit.html.md]: I'll update it to be more specific ie "your tech lead"

File: source/manual/alerts/varnish-port-not-responding.html.md [hunk 2]
```diff
@@ -39,17 +39,24 @@ $ fab $environment -H cache-3.router do:'ps -ef | grep [/]usr/sbin/varnishd'
...
+will fail where it succeeds below):
 
 ```
 $ fab $environment -H cache-3.router do:'/usr/lib/nagios/plugins/check_procs -c 1:1 -C 'varnishd' -p `< /var/run/varnishd.pid`''
 [cache-3.router] out: PROCS OK: 1 process with command name 'varnishd', PPID = 8273
 ```
 
+### Related Kibana query
+
+You can view the number of 5xx errors by [logging into Logit](/manual/logit/html),
```
Comment (tijmenb) [inline source/manual/alerts/varnish-port-not-responding.html.md]: Typo: `/manual/logit/html` -> `/manual/logit.html`

**Comment by surminus** [comment]
Fixed up!

**Pull request merged**